lib/osf-components/addon/components/search-page/template.hbs

```diff
@@ -200,6 +200,19 @@ as |layout|>
                                     </LinkTo>
                                 </li>
                             {{/each}}
+                            <div
+                                data-test-topbar-sort-dropdown
+                                local-class='sort-dropdown'
+                            >
```

Comment:
A `ul` should only have `li` as direct children, so this should move outside of the `ul`. There is also an `((#if @showResourceTypeFilter}}` statement on line 183, so this sort dropdown won't show up if we are on the branded registries or preprints page where we set `@showResourceTypeFilter={{false}}` (not an immediate concern for this release, but will need to move for the future release)

Reply:
Ok, this div structure is now correct. Thank you.